app/views/posts/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Post Show Works</h1>
\ No newline at end of file
+<section class="container-fluid mt-3">
+    <div class="card border border-black">
+        <div class"card-body">
+            <div class="card-title d-flex justify-content-between">
+                <h5 class="card-title"><%= @post.title %> by <%= @post.author.name %></h5>
+                <h6 class="card-title">
+                <small class="d-flex flex-wrap align-items-center">
+                    <%= form_with model: @like, url: user_post_likes_path(user_id: @post.author.id, post_id: @post.id), local: true do |form| %>
+                        <button type="submit" class="fa fa-heart-o pe-3 border border-0 btn"></button>
+                    <% end %>
+                    <span class="card-title pe-3">Comments: <%= @post.comments_counter %></span>
+                    <span class="card-title pe-3">Likes: <%= @post.likes_counter %></span>
+                </small>
+                </h6>
+            </div>
+            <p class="card-text"><%= @post.text %></p>
+        </div>
+    </div>
+    <div class="card border border-black">
+        <%= render "shared/comment", post: @post %>
+    </div>
+</section> 
```

Comment:
- [ ] Please, kindly ensure that all the comments (not recent comments) are displayed in the post_show page as illustrated in the wireframe. ℹ️ 
- [ ] For good user experience please ensure that the `like text and like icon` are wrapped up together. It was only the `like` icon that was functional, this can be misleading because the icon is far away to the text.

<img width="211" alt="6" src="https://github.com/SFarida/rails_blog_application/assets/103386183/2df159fd-a3fc-49de-b5e5-83240f1a17d2">

<img width="952" alt="6" src="https://github.com/SFarida/rails_blog_application/assets/103386183/eafe0ae1-e2e1-4a8f-a107-5b0885ae7c99">

Reply:
Hi @OyePriscilla  thanks for the remarks my next commit fixes the issues you mentionned